### PR TITLE
Issue #84

### PR DESCRIPTION
**Pull request by AnshSancheti**
Adding ability to fast forward and rewind by holding next and previous song buttons

### CODE REVIEW

File: app/src/main/java/com/poupa/vinylmusicplayer/helper/PrevNextButtonOnTouchListener.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import android.view.MotionEvent;
+import android.view.View;
+import android.os.Handler;
+import android.view.View.OnClickListener;
+
+public class PrevNextButtonOnTouchListener implements View.OnTouchListener {
+    private Handler handler = new Handler();
+
+    private int initialInterval;
+    private final int normalInterval;
```
Comment (soncaokim) [inline app/src/main/java/com/poupa/vinylmusicplayer/helper/PrevNextButtonOnTouchListener.java:12]: Can you please rename these variables to make the usage explicit (i.e. skipTriggerIntervalMillis)

File: app/src/main/java/com/poupa/vinylmusicplayer/ui/fragments/player/card/CardPlayerPlaybackControlsFragment.java [hunk 2]
```diff
@@ -49,6 +51,10 @@
 
     private MusicProgressViewUpdateHelper progressViewUpdateHelper;
 
+    private final int INITIAL_INTERVAL_MILLI = 1000;
```
Comment (soncaokim) [inline app/src/main/java/com/poupa/vinylmusicplayer/ui/fragments/player/card/CardPlayerPlaybackControlsFragment.java:54]: Is that possible to move these defaults to the previous class, i.e. keep them close to where they make sense.

Or rename them to clarify the usage (i,e. SKIP_TRIGGER_INTERVAL_MILLIS)

**Comment by soncaokim** [comment]
This proposes a fix for https://github.com/AdrienPoupa/VinylMusicPlayer/issues/84
(for some reason, Github is not picking up the commit comment, so this PR comment should help to link issues and PR)

File: app/src/main/java/com/poupa/vinylmusicplayer/helper/PrevNextButtonOnTouchListener.java (hunk 1 above)
Comment (AnshSancheti) [reply app/src/main/java/com/poupa/vinylmusicplayer/helper/PrevNextButtonOnTouchListener.java]: Resolved in the most recent commit

File: app/src/main/java/com/poupa/vinylmusicplayer/ui/fragments/player/card/CardPlayerPlaybackControlsFragment.java (hunk 2 above)
Comment (AnshSancheti) [reply app/src/main/java/com/poupa/vinylmusicplayer/ui/fragments/player/card/CardPlayerPlaybackControlsFragment.java]: Resolved in the most recent commit

**Review by soncaokim — APPROVED** [review]
Lgtm.

**Pull request merged**